File: osis/labs/LR-1-2/block-2/main.py
sum = int(input('Введите вашу сумму денег: '))
total_bottles = 0

buy_bottles = sum // 45
total_bottles += buy_bottles
sum = sum % 45 + buy_bottles * 20
if sum >= 45:
    total_bottles += 1

print(f"Куплено бутылок: {total_bottles}")

File: osis/labs/LR-1-2/block-2/main.pas
var sum, total_bottles, buy_bottles: integer;
begin
  write('Введите вашу сумму денег: ');
  read(sum);
  buy_bottles:= sum div 45;
  total_bottles:= total_bottles+buy_bottles;
  sum:=sum mod 45 + buy_bottles * 20;
  if sum >= 45 then total_bottles:= total_bottles + 1;
  Writeln('Куплено бутылок:', total_bottles);
end.